WalletWasabi/WabiSabi/Backend/DoSPrevention/Warden.cs

```diff
@@ -50,47 +61,18 @@ private static Prison DeserializePrison(string prisonFilePath)
 			}
 		}
 
-		var prison = new Prison(inmates);
-
-		var (noted, banned, longBanned) = prison.CountInmates();
-		if (noted > 0)
-		{
-			Logger.LogInfo($"{noted} noted UTXOs are found in prison.");
-		}
-
-		if (banned > 0)
-		{
-			Logger.LogInfo($"{banned} banned UTXOs are found in prison.");
-		}
-
-		if (longBanned > 0)
-		{
-			Logger.LogInfo($"{longBanned} long-banned UTXOs are found in prison.");
-		}
-
-		return prison;
+		return new Prison(config, coinjoinIdStore, inmates, channelWriter);
```

Comment:
IDK, I didn't change that but I think makes a lot of sense to load what we can, yes. Also, the only operation is appendline so, it could be that if the file is broken (it never  happened btw) it could be broken at the end so, most of the file can be recovered. Anyway, if that's something important, we should delete the file and write everything we have loaded back.